hardware/opas/TOPAS/TOPAS-800.py

```diff
@@ -15,8 +15,8 @@
 class Driver(BaseDriver):
     
     def __init__(self, *args, **kwargs):
-        #self.motor_names = ['Crystal', 'Amplifier', 'Grating', 'NDFG_Crystal', 'NDFG_Mirror', 'NDFG_Delay']
-        self.motor_names = ['0', '1', '2', '3', '4', '5']
+        self.motor_names = ['Crystal', 'Amplifier', 'Grating', 'NDFG_Crystal', 'NDFG_Mirror', 'NDFG_Delay']
+        #self.motor_names = ['0', '1', '2', '3', '4', '5']
```

Comment:
Thanks to wright-group/attune#78, we can refer to the motors by name...

The names _do_ need to match what WinTopas names the motors, meaning these ones probably need spaces instead of underscores (yes, that breaks using dot notation.... alternatively, we can change the name in WinTopas)

Reply:
Update, I have changed names in WinTopas4 to match the underscore names